recipe/shelf.py

```diff
@@ -427,30 +432,55 @@ class Meta:
         select_from = None
         engine = None
         ingredient_order = []
+        metadata = None
```

Comment:
shelf.Meta.metadata is for the recipe from config?

Reply:
oh, no. I think I misunderstood some other code as passing metadata to the Shelf constructor. I'll get rid of this :)

Reply:
wait, no. I added it here because the code was already setting self.Meta.metadata